bin/pc2installsandbox

```diff
@@ -72,11 +80,11 @@ fi
 echo ...done.
 
 # First, remove any existing remnants
-for dir in `ls $PC2_CGROUP_PATH/sandbox* 2>/dev/null`
+for dir in `ls -d $PC2_CGROUP_PATH/sandbox* 2>/dev/null`
 do
 	if test -d "$dir"
 	then
-		DEBUG echo Removing existing $dir
+		$DEBUG echo Removing existing $dir
 		rmdir "$dir"
```

Comment:
Can you assume that $dir is empty?   (Cautiously) use rm -fr $dir 
to remove dir and any contents.

Reply:
In this instance, yes, $dir "can not" be empty, or the 'for()' statement would not execute.

Reply:
Also, can't use "rm" on /sys/fs/cgroup, only 'rmdir', since they are not real files and "rm" does not understand the /sys/fs/cgroup filesystem.

Reply:
approved/understood rmdir effectively does rm -fr on sys/fs/cgroup file system.